tests/testthat/test-pin.R

```diff
@@ -1,4 +1,11 @@
 skip_if_not_installed("filelock")
+skip_if_not_installed("webfakes")
+
+httpbin <- local_httpbin_app()
+httpbin_port <- httpbin$get_port()
+redact_port <- function(snapshot) {
+  snapshot <- gsub(httpbin_port, "<port>", snapshot, fixed = TRUE)
+}
```

Comment:
In the future, I'll need to put this same bit in the `board_url()` test file. I couldn't put this in `setup.R` because webfakes is only in Suggests and it is very helpful to keep that "hard" GH action passing for pins.